src/main/java/com/forgeessentials/scripting/ModuleScripting.java

```diff
@@ -199,14 +199,14 @@ public void createDefaultPatternCommands()
         {
             cmd = new PatternCommand("god", "/god on|off [player]", null);
             cmd.getPatterns().put("on @p", Arrays.asList(new String[] { "permcheck fe.commands.god.others", //
-                    "permset user @0 deny fe.protection.damageby.*", "$*/heal @player", "echo God mode turned ON for @0" }));
+                    "permset user @0 deny fe.protection.damageby.*", "$*/heal @0", "echo God mode turned ON for @0" }));
             cmd.getPatterns()
                     .put("off @p",
                             Arrays.asList(new String[] { //
-                                    "permcheck fe.commands.god.others", "permset user %@ clear fe.protection.damageby.*",
+                                    "permcheck fe.commands.god.others", "permset user @0 clear fe.protection.damageby.*",
                                     "echo God mode turned OFF for @0", }));
             cmd.getPatterns().put("on", Arrays.asList(new String[] { //
-                    "permcheck fe.commands.god", "permset user @player deny fe.protection.damageby.*", "$*/heal", "echo God mode ON", }));
+                    "permcheck fe.commands.god", "permset user @player deny fe.protection.damageby.*", "$*/heal @player", "echo God mode ON", }));
```

Comment:
No, this is @player because there is no @0 in this pattern.  So this pattern heals the sender because the sender is requesting to god themselves.